pkg/kubernetes/informers.go

```diff
@@ -128,7 +149,7 @@ func (im *InformerManager) GetPVCLister() corelisters.PersistentVolumeClaimListe
 }
 
 // GetConfigMapLister returns ConfigMap Lister for the calling informer manager
-func (im *InformerManager) GetConfigMapLister() corelisters.ConfigMapLister {
+func (im *InformerManager) GetConfigMapLister(namespace string) corelisters.ConfigMapLister {
```

Comment:
Are we using the `namespace` argument anywhere?

Reply:
Good catch. Leaving this code unchanged.